enigma-principal/app/src/boot_network/principal_manager.rs

```diff
@@ -77,20 +78,20 @@ impl ReportManager {
     pub fn get_registration_params(&self) -> Result<RegistrationParams, Error> {
         let signing_address = self.get_signing_address()?;
         let sim_mode = option_env!("SGX_MODE").unwrap_or_default();
-        println!("Using SGX_MODE: {:?}", sim_mode);
-        println!("Fetching quote with SPID: {:?}", self.config.spid);
+        info!("Using SGX_MODE: {:?}", sim_mode);
```

Comment:
I am not sure under what conditions should the information of `get_registration_params` be printed. @lacabra, what do you think about this change?

Reply:
I would argue for Verbosity L2 (-vv), even L1 since that info is very useful, but in an upcoming change to P2P, that info should be displayed upon request from P2P, so the discussion should be moved there. But for now L1 or L2, if that makes sense. Thx!